internal/service/rds/cluster.go

```diff
@@ -237,10 +237,30 @@ func ResourceCluster() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: verify.ValidARN,
 			},
+			"manage_master_user_password": {
+				Type:          schema.TypeBool,
+				Optional:      true,
+				Default:       false,
+				ConflictsWith: []string{"master_password"},
+			},
+			"master_user_secret_kms_key_id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: verify.ValidARN,
+			},
+			//lintignore:S019
```

Comment:
The S019 analyzer reports cases of schemas including `Computed: false`,`Optional: false`, or `Required: false` which are extraneous. Removing `Optional: false` and `Required: false` would remove the need to ignore the linting rule.

Reference: [https://github.com/hashicorp/terraform-provider-aws/tree/main/.ci/providerlint/vendor/github.com/bflad/tfproviderlint/passes/S019](https://github.com/hashicorp/terraform-provider-aws/tree/main/.ci/providerlint/vendor/github.com/bflad/tfproviderlint/passes/S019)